cmd/managementd/main.go

```diff
@@ -68,14 +68,18 @@ func main() {
 	router.HandleFunc("/disk-memory", managementinterface.DiskMemoryHandler).Methods("GET")
 	router.HandleFunc("/location", managementinterface.LocationHandler).Methods("GET", "POST") // Form to view and/or set location manually.
 	router.HandleFunc("/clock", managementinterface.TimeHandler).Methods("GET", "POST")        // Form to view and/or adjust time settings.
-	router.HandleFunc("/about", managementinterface.AboutHandler).Methods("GET")
+	// router.HandleFunc("/about", managementinterface.AboutHandler).Methods("GET")
+
 	router.HandleFunc("/advanced", managementinterface.AdvancedMenuHandler).Methods("GET")
 	router.HandleFunc("/camera", managementinterface.CameraHandler).Methods("GET")
 	router.HandleFunc("/camera/snapshot", managementinterface.CameraSnapshot).Methods("GET")
 	router.HandleFunc("/rename", managementinterface.Rename).Methods("GET")
 
 	// API
 	apiObj := api.NewAPI(config.CPTVDir)
+	router.HandleFunc("/about", func(w http.ResponseWriter, r *http.Request) {
+		managementinterface.AboutHandler(w, r, apiObj)
+	}).Methods("GET")
```

Comment:
There's no need to pass apiObj in (it's not being used, see below). You can delete this and uncomment the line above.